app/static/js/darkmode.js

```diff
@@ -0,0 +1,29 @@
+const root = document.documentElement
+window.addEventListener('DOMContentLoaded', (event) => {
+    const button = document.querySelector('#darkmode');
+    console.log('loaded')
+    let isDarkMode = (localStorage.getItem('isDarkMode') == "true");
+    setThemeMode(isDarkMode, button);
+    button.addEventListener('click', event => {
+    isDarkMode = !isDarkMode;
+    localStorage.setItem('isDarkMode', isDarkMode);
+    setThemeMode(isDarkMode, button);
+    });
+});
+
+function changeIcon(mode) {
+    const modeButton = document.querySelector('#darkmode');
+    modeButton.name = mode;
+}
+
+function setThemeMode(isDarkMode, button) {
```

Comment:
Prefer passing `root` to this function.
It's better to have standalone functions which doesn't count on global variables.